src/Parser/Paladin/Retribution/Modules/Abilities.js

```diff
@@ -60,21 +60,11 @@ class Abilities extends CoreAbilities {
           recommendedEfficiency: 0.9,
         },
       },
-      {
-        spell: SPELLS.HOLY_WRATH_TALENT,
-        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-        cooldown: 180,
-        isOnGCD: true,
-        enabled: combatant.hasTalent(SPELLS.HOLY_WRATH_TALENT.id),
-        castEfficiency: {
-          suggestion: true,
-        },
-      },
       {
         spell: SPELLS.CRUSADER_STRIKE,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
         charges: 2,
-        cooldown: haste => 3.5 / (1 + haste),
+        cooldown: haste => 6 * 0.85 / (1 + haste),
```

Comment:
Instead of having two entries for Crusader strike depending on talent choice you could instead set the cooldown based on the talent:
`cooldown: haste => (combatant.hasTalent(SPELLS.THE_FIRES_OF_JUSTICE_TALENT.id) ? 5 : 6) / (1 + haste),`
btw 6*0.85 is 5.1 and not 5, idk if that's intentional?

Reply:
Will change this to one entry. Blizzard changed the talent to reduce the cooldown by 15% rather than 1 second, so 5.1 seconds is intentional.